src/pages/todo-app.jsx

```diff
@@ -1,32 +1,76 @@
-import React, { useEffect } from 'react'
-import { isLoggedIn, getCurrentUser, logOut } from "../utils"
+import React, { useEffect, useState } from 'react'
+import { isLoggedIn, getCurrentUser, logOut, fetch } from "../utils"
 import { useHistory } from 'react-router-dom'
 import person from '../images/person.svg'
 
 
 
+ const url = "https://bee-todo-app.herokuapp.com/todos"
 
-
+const accessToken = localStorage.getItem('accessToken')
 const TodoApp = () => {
     const history = useHistory()
-
+    const [ value, setValue ] = useState("")
+    const [ todo, setTodo ] = useState({ userId: getCurrentUser().id, description: value})
+    const [ tasks, setTasks ] = useState([])
+    const [ skip, setSkip ] = useState(0)
+    const [ limit, setLimit ] = useState(0)
+    const [ total, setTotal ] = useState(0)
 
     useEffect(() => {
-        if (isLoggedIn() === true) {
-            history.push("/dashboard")
-        } else {
+        if (isLoggedIn() === false) {
             history.push("/login")
+            return 
         }
+        history.push("/dashboard")
     }, [history])
+    
+    useEffect(()=> {
+        const fetchTasksEndpoint = `https://bee-todo-app.herokuapp.com/todos?userId=${getCurrentUser()?.id}&$skip=${skip}`
+        fetch(fetchTasksEndpoint, 'GET', null,  { 'Authorization': `Bearer ${accessToken}` })
+        .then((response)=> {
+          setTasks(tasks.concat(response.data.data))
+          setSkip(response.data.skip)
+          setLimit(response.data.limit)
+          setTotal(response.data.total)
+       }).catch((error) => {
+           console.log(error)
+       })
+    },[skip])
 
     const showInput = () => {
         let input = document.getElementById("input")
         input.classList.add("show-input")
     }
 
+    const changeValue = (e) => {
+        setValue(e.target.value)
+    }
+
     const addItem = () => {
+        todo.description = value
+        const newDescription = todo.description
+       setTodo({...todo}, newDescription)
+        fetch(url, 'POST', todo, { 'Authorization': `Bearer ${accessToken}` })
+         .then((response)=> {
+        }).catch((error) => {
+            console.log(error)
+        })
         let input = document.getElementById("input")
         input.classList.remove("show-input")
```

Comment:
we do not do this in ReactJS. we don't manually manipulate the DOM because it messes with React's virtual dom. Instead what you can do is create a state for the className of that input and then when you call the `addItem` function you can change the value of the className from `show-input` to "". Then pass the className into the input